openapi-core/src/main/java/io/ballerina/openapi/core/service/ServiceGenerationHandler.java

```diff
@@ -0,0 +1,70 @@
+package io.ballerina.openapi.core.service;
```

Comment:
missing license header

Reply:
This is a  temporary branch that we have to setup for easy collaboration. Once all files are correctly finalised, we will go through all and fix the docs, licenses etc. Hope that is fine